acceptance/help_test.go

```diff
@@ -13,19 +13,20 @@ const GLOBAL_USAGE = `ॐ
 om helps you interact with an Ops Manager
 
 Usage: om [options] <command> [<args>]
-  --client-id, -c, OM_CLIENT_ID                          string  Client ID for the Ops Manager VM (not required for unauthenticated commands)
-  --client-secret, -s, OM_CLIENT_SECRET                  string  Client Secret for the Ops Manager VM (not required for unauthenticated commands)
-  --connect-timeout, -o, OM_CONNECT_TIMEOUT              int     timeout in seconds to make TCP connections (default: 10)
-  --decryption-passphrase, -d, OM_DECRYPTION_PASSPHRASE  string  Passphrase to decrypt the installation if the Ops Manager VM has been rebooted (optional for most commands)
-  --env, -e                                              string  env file with login credentials
-  --help, -h                                             bool    prints this usage information (default: false)
-  --password, -p, OM_PASSWORD                            string  admin password for the Ops Manager VM (not required for unauthenticated commands)
-  --request-timeout, -r, OM_REQUEST_TIMEOUT              int     timeout in seconds for HTTP requests to Ops Manager (default: 1800)
-  --skip-ssl-validation, -k, OM_SKIP_SSL_VALIDATION      bool    skip ssl certificate validation during http requests (default: false)
-  --target, -t, OM_TARGET                                string  location of the Ops Manager VM
-  --trace, -tr, OM_TRACE                                 bool    prints HTTP requests and response payloads
-  --username, -u, OM_USERNAME                            string  admin username for the Ops Manager VM (not required for unauthenticated commands)
-  --version, -v                                          bool    prints the om release version (default: false)
+  --client-id, -c, OM_CLIENT_ID                            string  Client ID for the Ops Manager VM (not required for unauthenticated commands)
+  --client-secret, -s, OM_CLIENT_SECRET                    string  Client Secret for the Ops Manager VM (not required for unauthenticated commands)
+  --connect-timeout, -o, OM_CONNECT_TIMEOUT                int     timeout in seconds to make TCP connections (default: 10)
+  --decryption-passphrase, -d, OM_DECRYPTION_PASSPHRASE    string  Passphrase to decrypt the installation if the Ops Manager VM has been rebooted (optional for most commands)
+  --env, -e                                                string  env file with login credentials
+  --help, -h                                               bool    prints this usage information (default: false)
+  --ignore-verifier-warnings, OM_IGNORE_VERIFIER_WARNINGS  bool    continues working even if there are verifier warnings present (default: false)
```

Comment:
Can we make this description something more like: 
```
 option to ignore verifier warnings. NOT RECOMMENDED UNLESS DISABLED IN OPS MANAGER (default: false) 
```

The reason for this is that we do not want users to use this flag unless they absolutely know what they are doing. We want to protect people from themselves.

Reply:
Sounds good.